edx_event_bus_kafka/internal/consumer.py

```diff
@@ -381,6 +370,31 @@ def emit_signals_from_message(self, msg):
         if AUDIT_LOGGING_ENABLED.is_enabled():
             logger.info('Message from Kafka processed successfully')
 
+    def deserialize_message_value(self, msg):
+        """
+        Deserialize an Avro message value
+
+        Arguments:
+            msg (Message): the raw message from the consumer
+
+        Returns:
+            The deserialized message value
+        """
+        schema_registry_client = get_schema_registry_client()
```

Comment:
I believe this can be lifted to `__init__` rather than being created each time.

Reply:
Moved it to module initiation

Reply:
Nevermind, moving it to instance initiation because doing it at the top level of the module does weird things to tests. We could just have a test.py environment file but I would put that in a separate PR.